proposals/2020-11-stargate-upgrade/README.md

```diff
@@ -0,0 +1,133 @@
+
+Integration testing continued through the release of the Cosmos SDK v0.40.0-rc2 that makes up the Cosmos-hub Stargate testnet tag for a simulated upgrade of the CosmosHub-3 to CosmosHub-4. Also, the AiB team continues to deliver simulation testing of Cosmos Stargate to ensure that any possible chain bug issues can be detected and documented.
+
+We are confident that we have and continue to achieve increased ecosystem engagement with the Stargate testnet.
+
+### 2. Critical Partner Support
+
+We delivered critical partner support to leading ATOM exchanges. Out of the exchange support efforts, we documented the [IBC Readiness Matrix](https://github.com/cosmosdevs/stargate/blob/master/ibc_readiness_matrix.md) that outlines the levels of readiness that exchanges may select as they upgrade to Cosmos Stargate.
+
+We also completed and delivered a completed legacy Amino Audit, [AminoRest & You](https://github.com/cosmosdevs/stargate/blob/master/audit.md). The core finding of this audit was that changes to the underlying structs result in an interface that is close to the prior version allowing legacy queries to return valid data.
+
+### 3. Documentation and Self-Certification
+
+Documentation was a success for the Cosmos Stargate effort. The upgrades with the most breaking changes such as legacy Amino have a complete audit with documentation on exceptions for blockchain API interfaces.
+
+Most exchanges and wallets have taken self-certification on directly. Our team continues to provide real-time support on multiple partner slack channels and on the Cosmos [#stargate Discord](https://discord.gg/W8trcGV) channel.
+
+### Cosmos Stargate Integration Success
+
+Cosmos Stargate integration success with exchanges and wallet providers reflects the quality of the code developed by the various Cosmos teams.
+
+## Prop 29
+
+This upgrade also implements the fund recovery procedure defined in proposal 29. The code is [here](https://github.com/cosmos/gaia/blob/main/app/prop29.go). As requested during the discussion of the governance proposal, the migration and fund recovery procedure verifies all signatures. This code was tested during the cosmoshub-3 simulated upgrade testnet.ƒ
+
+## Stargate Upgrade Steps
+
+------------
+The upgrade steps for the simulated upgrade of the current Cosmoshub mainnet to Cosmos Stargate includes the following steps:
+
+This section is with the current `gaia 2.0.*` implementation.
+
+  1. Validators should set their `gaia` with a halt time of `1611813600`ie. 06:00 UTC on Jan 28th in Unix time.
+
+  2. Validators should then export the current cosmos state with `gaiad export > cosmoshub-3-export.json`
+
+  3. Validators should determine the height of the last block as `last-cosmoshub-3_block`.
+
+  4. Validators should back up their `.gaiad` directory.
+
+This section is with the upgrade `gaia 3.0.*` implemenataion.
```

Comment:
Should we be more explicit about doing the installation at this point, or does every validator know what this means?

Reply:
yeah it's outlined on the release notes